src/source/lexbase.d

```diff
@@ -564,6 +564,25 @@ unittest {
 		assert(t.error.toString(context) == error);
 	}
 
+	auto checkIncompleteInput(string s, TokenType expected) {
+		auto lex = makeTestLexer(s).withComments(true);
+
+		import source.parserutil;
+		import source.exception;
+		lex.match(TokenType.Begin);
+
+		try {
+			lex.match(expected);
```

Comment:
It looks like this is testing the match function, so why is this in lexbase? It does looks like it belong in parserutil.

Reply:
moved